packages/cactus-test-plugin-ledger-connector-besu/src/test/typescript/integration/plugin-validator-besu/get-balance-endpoint.test.ts

```diff
@@ -0,0 +1,176 @@
+    rpcApiWsHost,
+    pluginRegistry,
+    logLevel,
+  };
+  const pluginValidatorBesu = new PluginLedgerConnectorBesu(options);
+
+  // 4. Create the API Server object that we embed in this test
+  const configService = new ConfigService();
+  const apiServerOptions = configService.newExampleConfig();
+  apiServerOptions.authorizationProtocol = AuthorizationProtocol.NONE;
+  apiServerOptions.configFile = "";
+  apiServerOptions.apiCorsDomainCsv = "*";
+  apiServerOptions.apiPort = addressInfo1.port;
+  apiServerOptions.cockpitPort = 0;
+  apiServerOptions.apiTlsEnabled = false;
+  const config = configService.newExampleConfigConvict(apiServerOptions);
+
+  pluginRegistry.add(pluginValidatorBesu);
+
+  const apiServer = new ApiServer({
+    httpServerApi: httpServer1,
+    config: config.getProperties(),
+    pluginRegistry,
+  });
+
+  // 5. make sure the API server is shut down when the testing if finished.
+  test.onFinish(() => apiServer.shutdown());
+
+  // 6. Start the API server which is now listening on port A and it's healthcheck works through the main SDK
+  await apiServer.start();
+
+  // 7. Instantiate the main SDK dynamically with whatever port the API server ended up bound to (port 0)
+  t.comment(`AddressInfo: ${JSON.stringify(addressInfo1)}`);
+
+  const web3Provider = new Web3.providers.HttpProvider(rpcApiHttpHost);
+  const web3 = new Web3(web3Provider);
+  const web3Eea: IWeb3InstanceExtended = EEAClient(web3, 2018);
+
+  const orionKeyPair = await besuTestLedger.getOrionKeyPair();
+  const besuKeyPair = await besuTestLedger.getBesuKeyPair();
+
+  const besuPrivateKey = besuKeyPair.privateKey.toLowerCase().startsWith("0x")
+    ? besuKeyPair.privateKey.substring(2)
+    : besuKeyPair.privateKey; // besu node's private key
+
+  const contractOptions = {
+    data: `0x123`,
+    // privateFrom : Orion public key of the sender.
+    privateFrom: orionKeyPair.publicKey,
+    // privateFor : Orion public keys of recipients or privacyGroupId: Privacy group to receive the transaction
+    privateFor: [orionKeyPair.publicKey],
+    // privateKey: Ethereum private key with which to sign the transaction.
+    privateKey: besuPrivateKey,
+  };
+
+  const transactionHash = await web3Eea.eea.sendRawTransaction(contractOptions);
+  await web3.eth.getTransaction(transactionHash);
+
+  /*
+  const transaction = await web3.eth.getTransaction(transactionHash);
+  const singData = jsObjectSigner.sign(transaction.input);
+  const signDataHex = Buffer.from(singData).toString("hex");
+  */
+
+  const request: GetBalanceV1Request = {
+    //is it suppose to be GetBalanceV1Request?
+    address: testAccount.address,
+  };
+
+  const configuration = new BesuApiClientOptions({ basePath: node1Host });
+  const api = new BesuApiClient(configuration);
+
+  // Test for 200 valid response test case
+  const res = await api.getbalanceV1(request);
+  t.ok(res, "API response object is truthy");
+  t.true(res.data.balance, "Response is String ok");
```

Comment:
@Leeyoungone You are missing a `typeof` here, you want to assert that the typeof res.data.balance === "string" is true